src/TTreeWidget.h

```diff
@@ -60,6 +61,20 @@ class TTreeWidget : public QTreeWidget
     void setIsKeyTree();
     void beginInsertRows(const QModelIndex& parent, int first, int last);
     void getAllChildren(QTreeWidgetItem*, QList<QTreeWidgetItem*>&);
+    int m_childID;
+    int m_oldParentID;
+    int m_newParentID;
+    int m_parentPosition;
+    int m_childPosition;
+    int m_PrevParentPosition;
+    int m_PrevChildPosition;
+    QModelIndex m_parent;
+    QUndoStack* undoStack;
```

Comment:
For pointers the prefix we use is `mp` - and we initialise them (if they are not "smart" ones that initialise themselves) to make things go :boom: quicker if they get used without being set to something:
```suggestion
    QUndoStack* mpUndoStack = nullptr;
```